src/constraint_solver/io.cc

```diff
@@ -180,6 +188,16 @@ class FirstPassVisitor : public ModelVisitor {
   }
 
  private:
+  //// TODO: TBD: not sure we are confusing terms with IntExpr, losing IntVar-iness, nevermind Boolean, etc...
```

Comment:
@lperron Laurent, I ran a unit test that [proves otherwise](http://github.com/google/or-tools/issues/358). Furthermore, the fact that ``IntVar``'s are being *delegated* to ``IntExpr``, underscores this fact. So... What would it take ***not*** to delegate to ``IntExpr``?